libs/nil_core/include/zkevm_framework/core/utils.hpp

```diff
@@ -0,0 +1,122 @@
+#ifndef ZKEMV_FRAMEWORK_LIBS_NIL_CORE_INCLUDE_ZKEVM_FRAMEWORK_CORE_UTILS_HPP_
```

Comment:
Why we actually need read/write_bytes?

Reply:
Well, in fact we don't. These functions were useful before when we had stream read/write interfaces. Now they may be useful if you will need to write `Bytes<N>`-based types into stream. 

But we can safely remove them, it's up to you, I just didn't want to remove anything which may be useful.